### PR TITLE
update completion-mode

### PR DESCRIPTION
**Pull request by LittleAaron**
更新完成模式，同步英文文档

### CODE REVIEW

**Comment by LittleAaron** [comment]
cla signed

**Comment by RA489** [comment]
> cla signed

/check-cla

**Review by Arhell — COMMENTED** [review]
/check-cla

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -360,13 +366,14 @@ Jobs with _fixed completion count_ - that is, jobs that have non null
...
-  当每个索引都对应一个完成完成的 Pod 时，Job 被认为是已完成的。
-  关于如何使用这种模式的更多信息，可参阅
-  [用带索引的 Job 执行基于静态任务分配的并行处理](/zh/docs/tasks/job/indexed-parallel-processing-static/)。
-  需要注意的是，对同一索引值可能被启动的 Pod 不止一个，尽管这种情况很少发生。
-  这时，只有一个会被记入完成计数中。
+- `Indexed`：Job 的 Pod 会获得对应的完成索引，取值为 0 到 `.spec.completions-1`。该索引可通过三种机制获得：
+  - Pod注解 `batch.kubernetes.io/job-completion-index`.
+  - 作为Pod主机名的一部分, 遵循模式 `$(job-name)-$(index)`.
+    当你将Indexed Job和{{< glossary_tooltip term_id="Service" >}}一同使用时, 
+    Job中的 Pods 可以使用确定性主机名通过 DNS 相互寻址。
```
Comment (mayocream) [inline content/zh/docs/concepts/workloads/controllers/job.md:373]:
```suggestion
  - Pod 注解 `batch.kubernetes.io/job-completion-index`。
  - 作为 Pod 主机名的一部分，遵循模式 `$(job-name)-$(index)`。
    当你将带索引的 Job 和 {{< glossary_tooltip term_id="Service" >}} 一同使用时， 
    Job 中的 Pods 可以使用确定性主机名通过 DNS 相互寻址。
```

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 2]
```diff
@@ -360,13 +366,14 @@ Jobs with _fixed completion count_ - that is, jobs that have non null
...
+  - 从容器化任务的环境变量`JOB_COMPLETION_INDEX`。当每个索引都对应一个完成完成的 Pod 时，Job 被认为是已完成的。
```
Comment (mayocream) [inline content/zh/docs/concepts/workloads/controllers/job.md:374]:
```suggestion
  - 从容器化任务的环境变量 `JOB_COMPLETION_INDEX`。
    当每个索引都对应一个完成的 Pod 时，Job 被认为是已完成的。
```

File: content/zh/docs/concepts/workloads/controllers/job.md [hunk 3]
```diff
@@ -360,13 +366,14 @@ Jobs with _fixed completion count_ - that is, jobs that have non null
...
+    使用这种模式的更多信息，可参阅[用带索引的 Job 执行基于静态任务分配的并行处理](/zh/docs/tasks/job/indexed-parallel-processing-static/)。
+    需要注意的是，对同一索引值可能被启动的 Pod 不止一个，尽管这种情况很少发生。这时，只有一个会被记入完成计数中。
```
Comment (mayocream) [inline content/zh/docs/concepts/workloads/controllers/job.md:376]: 换行
```suggestion
    使用这种模式的更多信息，可参阅
    [用带索引的 Job 执行基于静态任务分配的并行处理](/zh/docs/tasks/job/indexed-parallel-processing-static/)。
    需要注意的是，对同一索引值可能被启动的 Pod 不止一个，尽管这种情况很少发生。
    这时，只有一个会被记入完成计数中。
```

**Comment by jimangel** [comment]
Hey there! @LittleAaron, looks like you haven't signed the CLA yet. Could I please have you do that? https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by reylejano** [comment]
Hi @LittleAaron , 
I'm closing this PR since the CLA has not been signed yet, this PR needs rebase, and there has been no traction on this PR for a month. Feel free to reopen this PR when you have signed the CLA and are ready to rebase this PR

/close

**Pull request closed without merging**